src/compiler/config/outputs/index.ts

```diff
@@ -22,6 +22,9 @@ export const validateOutputTargets = (config: d.Config, diagnostics: d.Diagnosti
       err.messageText = `Invalid outputTarget type "${
         outputTarget.type
       }". Valid outputTarget types include: ${VALID_TYPES.map((t) => `"${t}"`).join(', ')}`;
+    } else if (outputTarget.type === DIST_CUSTOM_ELEMENTS_BUNDLE) {
+      const warning = buildWarn(diagnostics);
```

Comment:
Can we add a `TODO` for the eventual removal of this code (when we remove the output target)? It looks like we have a ticket already defined for that work, so all I think we need is to add something to the effect of:
```suggestion
      // TODO(STENCIL-260): Remove this check when the 'dist-custom-elements-bundle' is removed
      const warning = buildWarn(diagnostics);
```